app/uk/gov/hmrc/gform/views/form/form.scala.html

```diff
@@ -124,7 +125,7 @@
       }
     }
 
-    @formWithoutCSRF(action = page.formAction, args = 'class -> "js-form", 'autocomplete -> "on", 'novalidate -> "novalidate", 'id -> "gf-form") {
+    @formWithCSRF(action = page.formAction, args = 'class -> "js-form", 'autocomplete -> "on", 'novalidate -> "novalidate", 'id -> "gf-form") {
```

Comment:
How come this is hardcoded to `formWithCSRF`? This will still send `csrfToken` to upscan which is going to fail. Or am I missing something?

Should this be managed by same condition as `formAction`:
```
if (fileUploadProviders.size === 1) formWithoutCSRF else formWithCSRF
```